client/my-sites/domain-tip/index.jsx

```diff
@@ -46,10 +46,21 @@ const DomainTip = React.createClass( {
 		return true;
 	},
 
+	domainUpgradeNudge() {
+		return (
+			<UpgradeNudge
+					title={ this.translate( 'Get a free Custom Domain' ) }
+					message={ this.translate( 'Custom domains are free when you upgrade to a Premium or Business plan.' ) }
+					feature={ FEATURE_CUSTOM_DOMAIN }
+					event="stats_insights_domain"
```

Comment:
Not sure if we want to hardcode this? Should we pass an event prop to domain tip? How would we differentiate different domain tips in different places?